### DISCUSSION TITLE
一些跨性别者，让我怀疑了我是否站在了正确的立场上

### DISCUSSION THREAD

**Discussion by Nofated095** [LGBT-CN/LGBTQIA-In-China#195] [General]
如题，我之前写过一篇关于跨性别者保护的一篇文章，里面提到了一些关于保护他们利益和生活的一些具体措施。
# 但是
# 我现在怀疑我是否站在了正确的立场上
# 我不知道为他们而付出的努力有没有意义
### 上述问题存在于 MtF 社区，当然不是每一个人，但是是目前我认为急需解决的问题

**Comment by Nofated095** [comment]
写的有一些激进，但是确实是我当时的想法

**Comment by gxres042** [comment]
并不是所有的 MtF 都会是这样子，就比如，和你居住在同一个城市的一位 MtF（因隐私问题我不在这里讲）。
但，说实在，会有这么一群 MtF 拿着公共资源搞这种 sexy 行为我是确实没想到，虽然 Twitter 上见到这种内容其实也挺正常的吧......
根据你所提供的信息进行分析，如果他们只是纯粹想要交配的话，那应该还能理解一下；如果真的是想当那些 sexy 的角色的话，真的没有 MtF 的必要了。

**Comment by Nofated095** [comment]
# 我说句难听的
# 某 些 MTF 根 本 就 不 需 要 什 么 保 护 吧
# 一个个恨不得将来就是上海名媛
# ~~也许一个15岁的臭傻逼关注平权本身就是一件傻逼事情，我就是个傻逼~~

**Comment by Nofated095** [comment]
我愿意支持那些能够从性转中获取精神慰藉的人们，但不代表我支持他们去做爱

**Comment by LeeZhihan** [comment]
> 都拿着国家物资去干援交。

順女，LES，我覺得你不適合來海外的社群，你適合用微博，拿著華爲用微博。

**Comment by kecrily** [comment]
Did you know that some people need to do things they don't necessarily like to earn the money they need to do SRS or take medication?

> 我愿意支持那些能够从性转中获取精神慰藉的人们，但不代表我支持他们去做爱

In addition sex is a basic human right

**Comment by Nofated095** [comment]
> > 都拿着国家物资去干援交。
> 
> 順女，LES，我覺得你不適合來海外的社群，你適合用微博，拿著華爲用微博。

我说的有一些部分是错误的，但是我反对说保护了她们，转手就通过一些形式挥霍这些保护，“援交”这个词可能用的不太准确，反正我认为是其中一种浪费的形式，但就算没有，也有很多其他被浪费了。

谢谢，我相信不用证明狗都不用微博和华为的结论。
![image](https://user-images.githubusercontent.com/49985975/178150998-5dd85a01-731a-4ab5-aeac-b8b708c2c541.png)

**Comment by Nofated095** [comment]
> Did you know that some people need to do things they don't necessarily like to earn the money they need to do SRS or take medication?
> 
> > 我愿意支持那些能够从性转中获取精神慰藉的人们，但不代表我支持他们去做爱
> 
> In addition sex is a basic human right

我清楚你说的情况，和我文章中所说的一样，确实说因为保护不够落地而导致很多证明和药物都开不了、买不了。
我说的色情方面存在一定的问题，但是在性跨中的的确确有存在浪费资源和保护的情况，~~我认为色情只是其中一种形式~~

我支持'sex is a basic human right'，对不起，可能从色情角度出发来说我的观点是错误的。

**Comment by Nofated095** [comment]
再解释一下
>我在 Twitter 上关注了一些 MtF，我本想关注关注他们的生活情况，好家伙，我可以用“色情无处不在”来形容 MtF 们，相当一大部分吃激素并且是性转的跨性别者几乎天天都在发有关于色情的推，甚至出现了很多 MtF 援交的情况。
我现在很难以接受这样的情况，我感觉对跨性别者保护到最后居然形成了一个色情群体，都拿着国家物资去干援交。
我现在很是怀疑之前我是否应该为他们而不断努力，我感觉自己的努力完全就是助长了性跨的色情。
如果性跨们都是这样的话，我虽然仍然支持平权，但我将坚决收回我的文章，并且将支持反跨，我的努力都被拿去做色情事业，我感觉所有努力全TM打了水漂

这一部分，从色情角度出发来看是存在问题的，但是我要表达的是存在 MtF 挥霍保护和资源的问题

**Comment by LeeZhihan** [comment]
> > > 都拿着国家物资去干援交。
> > 
> > 
> > 順女，LES，我覺得你不適合來海外的社群，你適合用微博，拿著華爲用微博。
> 
> 我说的有一些部分是错误的，但是我反对说保护了她们，转手就通过一些形式挥霍这些保护，“援交”这个词可能用的不太准确，反正我认为是其中一种浪费的形式，但就算没有，也有很多其他被浪费了。
> 
> 谢谢，我相信不用证明狗都不用微博和华为的结论。 ![image](https://user-images.githubusercontent.com/49985975/178150998-5dd85a01-731a-4ab5-aeac-b8b708c2c541.png)

MBA 2012 真沒必要秀了…

十五歲還是好好迎接初三吧

**Comment by Nofated095** [comment]
> MBA 2012

谢谢，并不说在秀，只是说一下狗都不用微博和华为
#184 中的签署和文章都是在中考结束后进行的。

**Comment by LeeZhihan** [comment]
> > MBA 2012
> 
> 谢谢，并不说在秀，只是说一下狗都不用微博和华为 #184 中的签署和文章都是在中考结束后进行的。

既然這樣，相信你也有時間聊聊吧

我很好奇你對於 MtF 的看法，你認爲，她們應該是怎麽樣的？

你認爲我們，我們 LES，又應該是如何的？

**Comment by Nofated095** [comment]
> > > MBA 2012
> > 
> > 
> > 谢谢，并不说在秀，只是说一下狗都不用微博和华为 #184 中的签署和文章都是在中考结束后进行的。
> 
> 既然這樣，相信你也有時間聊聊吧
> 
> 我很好奇你對於 MtF 的看法，你認爲，她們應該是怎麽樣的？
> 
> 你認爲我們，我們 LES，又應該是如何的？

谢谢你愿意给我聊
（对不起对于LES群体不是很有了解，可能说不了什么

我是希望国内的MtF可以通过多种途径的保护，从而获得真正跨过去的性别的所有权利，可以和我们正常的共享一切我们都享有的社会福利和资源。

迫于国内的环境，通过援交来获取检查的资金和药品购买，从援交的性质上我是无法接受的，但是我明白她们的目的是好的。

我的愿望是MtF们能像我们一样参与到积极的社会生活中，因为社会主义核心价值观在那里摆着，积极融入他才能获得更多公正待遇。

我认识和上述特征比较像的 MtF 们，她们平时和普通人一样可以去上街购物，我也很开心把她们当普通女孩子看待，我很高兴他们能从跨中获得精神慰藉。

**Comment by Nofated095** [comment]
> > > MBA 2012
> > 
> > 
> > 谢谢，并不说在秀，只是说一下狗都不用微博和华为 #184 中的签署和文章都是在中考结束后进行的。
> 
> 既然這樣，相信你也有時間聊聊吧
> 
> 我很好奇你對於 MtF 的看法，你認爲，她們應該是怎麽樣的？
> 
> 你認爲我們，我們 LES，又應該是如何的？

简单去看了一下 LES 的概念，大概有了一个理解

我觉得和 MtF 一样吧，我觉得在限度之内的sex是我完全能接受的，同样，尽管sex是human's right，我不支持发生为了纯粹金钱和利益的性关系，或者其他浪费社会资源和保护的行为。
如果 LES 和我做朋友，我也可以很容易接受。如果她们生活有困难，我也愿意帮助她们。

**Comment by LeeZhihan** [comment]
> > > > MBA 2012
> > > 
> > > 
> > > 谢谢，并不说在秀，只是说一下狗都不用微博和华为 #184 中的签署和文章都是在中考结束后进行的。
> > 
> > 
> > 既然這樣，相信你也有時間聊聊吧
> > 我很好奇你對於 MtF 的看法，你認爲，她們應該是怎麽樣的？
> > 你認爲我們，我們 LES，又應該是如何的？
> 
> 简单去看了一下 LES 的概念，大概有了一个理解
> 
> 我觉得和 MtF 一样吧，我觉得在限度之内的sex是我完全能接受的，同样，我不支持发生为了纯粹金钱和利益的性关系，尽管这是human's right
> 
> 如果 LES 和我做朋友，我也可以很容易接受。如果她们生活有困难，我也愿意帮助她们。

粗略看了下，可惜時間有些晚了 —— 明天是東城區期末考試的最後一天，放學後回復你。

晚安。

**Comment by Nofated095** [comment]
> > > > > MBA 2012
> > > > 
> > > > 
> > > > 谢谢，并不说在秀，只是说一下狗都不用微博和华为 #184 中的签署和文章都是在中考结束后进行的。
> > > 
> > > 
> > > 既然這樣，相信你也有時間聊聊吧
> > > 我很好奇你對於 MtF 的看法，你認爲，她們應該是怎麽樣的？
> > > 你認爲我們，我們 LES，又應該是如何的？
> > 
> > 
> > 简单去看了一下 LES 的概念，大概有了一个理解
> > 我觉得和 MtF 一样吧，我觉得在限度之内的sex是我完全能接受的，同样，我不支持发生为了纯粹金钱和利益的性关系，尽管这是human's right
> > 如果 LES 和我做朋友，我也可以很容易接受。如果她们生活有困难，我也愿意帮助她们。
> 
> 粗略看了下，可惜時間有些晚了 —— 明天是東城區期末考試的最後一天，放學後回復你。
> 
> 晚安。

谢谢，我也是北京东城区的。晚安

**Comment by Nofated095** [comment]
![image](https://user-images.githubusercontent.com/49985975/178187746-0616e53a-6d9d-489f-9cdc-c0cf36efe181.jpg)
![Uploading image.jpg…]()

**Comment by Nofated095** [comment]
我为我所有说过的话，以及不成熟的言论，以及咄咄逼人的态度，沉重地道歉，并对伤害到的 MtF 们抱歉。对不起，我撤回我说过的所有话。
我的行为看上去可能非常撒比，希望各位你们可以原谅，当然也许你们不会原谅。
再一次沉重地道歉，出现的损失是无法挽回的，我会尽量用自己的行为弥补

**Comment by Nofated095** [comment]
我重新表明一下立场：
支持 LGBT-CN，支持发展。
只是**以建议形式来建议各位跨性者（尤其是在中国大陆的）可以更加注意自己的行为是否符合中国的法律**
违反价值观没什么问题，但毕竟~~得益于~~中国的法律体系，最好还是安分守己一点吧（

**Comment by Nofated095** [comment]
如果可以的话，我希望可以重新签署。。。

**Comment by gxres042** [comment]
Maybe their behaviour is the reason that many people hate LGBTQIA.

I agree to keep following the law of Chinese Mainland, I will always disagree the people who said like "fuck your freedom" that have no any effective evidence.

Thanks for you to support LGBT-CN again.

**Comment by gxres042** [comment]
> 如果可以的话，我希望可以重新签署。。。

You can try to sign your name again, I will approve the change if not any problems.

**Comment by alex3236** [comment]
> Maybe their behaviour is the reason that many people hate LGBTQIA.
> 
> I agree to keep following the law of Chinese Mainland, I will always disagree the people who said like "fuck your freedom" that have no any effective evidence.
> 
> Thanks for you to support LGBT-CN again.

Can't agree with you more

**Comment by alex3236** [comment]
In fact I think this kind of topic should be discussed in Discussions not Issues as it has nothing to do with the repo itself. 🤔

**Comment by Nofated095** [comment]
> In fact I think this kind of topic should be discussed in Discussions not Issues as it has nothing to do with the repo itself. thinking

Good idea.
